errors/errors_test.go

```diff
@@ -0,0 +1,49 @@
+package errors
+
+import (
+	"errors"
+	"github.com/rs/zerolog"
+	"testing"
+)
+
+func TestE(t *testing.T) {
+
+	tests := []struct {
+		functionCall FCall
+		err          error
+		message      string
+		level        zerolog.Level
+	}{
+		{functionCall: "daemon.New", err: errors.New("configuration file not found"), message: "error on daemon.New function", level: zerolog.ErrorLevel},
+		{functionCall: "daemon.New", err: errors.New("No error informing ..."), message: "Daemon has been created", level: zerolog.InfoLevel},
+		{functionCall: "auth.NewAuthenticator", err: errors.New("Warning: "), message: "key is plain text", level: zerolog.WarnLevel},
+		{functionCall: "eventpool.RemoveEvent", err: errors.New("Debugging: "), message: "Removing event.", level: zerolog.DebugLevel},
+	}
+	for _, tt := range tests {
+		t.Run(tt.message, func(t *testing.T) {
+			if err := E(tt.functionCall, tt.err, tt.message); err.Error() != (tt.err.Error() + " " + tt.message) {
+				t.Errorf("E() error = %v", err)
+			}
+		})
+	}
+}
+
+func TestSeverity(t *testing.T) {
+	const functioncall FCall = "TestSeverity"
+	message := "error to test out severity of zerolog levels"
+	tests := []struct {
+		functionCall FCall
+		message      string
+		level        zerolog.Level
+	}{
+		{functionCall: functioncall, message: message, level: zerolog.ErrorLevel},
```

Comment:
We should also test the other Levels i think

Reply:
Since E returns error type, it is not quite obvious how to test it. I think, previous version of E function was good enough (with logzero instead of logrus), since it is just a helper function and do not need to wrap all arguments that user typed.

Reply:
But why set the log level in E then?